src/detections/table/body/data.js

```diff
@@ -0,0 +1,79 @@
+export const detections = [
+  {
```

Comment:
Let's also use different ids here, so that our redirects actually redirects to different pages.

Reply:
Done